python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/virtual-source-nat-vrfs.j2

```diff
@@ -10,5 +10,8 @@
 {%         if vrf.ip_address is arista.avd.defined %}
 ip address virtual source-nat vrf {{ vrf.name }} address {{ vrf.ip_address }}
 {%         endif %}
+{%         if vrf.ipv6_address is arista.avd.defined %}
```

Comment:
I think we should make 2 list for ipv4 and ipv6 because here we are sorting base vrf name but running config will have all ipv4 addresses then ipv6 addresses.
```
ip address virtual source-nat vrf TEST_01 address 1.1.1.1
ip address virtual source-nat vrf TEST_02 address 1.1.1.2
ip address virtual source-nat vrf TEST_04 address 1.1.1.3
ipv6 address virtual source-nat vrf TEST_03 address 2001:db8:85a3::8a2e:370:7334
ipv6 address virtual source-nat vrf TEST_04 address 2001:db8:85a3::8a2e:370:7335
```

Reply:
Fixed!